credsweeper/rules/config.yaml

```diff
@@ -709,6 +709,28 @@
     - src
     - doc
 
+- name: Bitbucket Client ID
+  severity: high
+  type: pattern
+  values:
+    - (^|[^0-9A-Za-z.$_/+-])(?P<value>[a-zA-Z0-9]{18}([a-zA-Z0-9]{14})?)([^0-9A-Za-z.$_/+-]|$)
+  filter_type: WeirdBase64Token
+  min_line_len: 18
+  usage_list:
+    - src
+    - doc
+
+- name: Bitbucket Client Secret
+  severity: medium
+  type: pattern
+  values:
+    - (^|[^0-9A-Za-z.$_/+-])(?P<value>([a-zA-Z0-9_-]{32}){1,2})([^0-9A-Za-z.$_/+-]|$)
+  filter_type: WeirdBase64Token
+  min_line_len: 32
+  usage_list:
+    - src
+    - doc
+
```

Comment:
I think those severity of those patterns should be `INFO`.
Could you please let me know why severities are `MEDIUM` and `HIGH`?

Reply:
Mistake after merge.

Reply:
Thank you for a quick fix!